doc/source/whatsnew/v1.3.3.rst

```diff
@@ -34,7 +34,7 @@ Bug fixes
 
 Other
 ~~~~~
--
+- Wheels are now provided for Python 3.10rc1 (:issue:`42946`)
```

Comment:
Do we want to actually provide wheels for the rc? Release is expected 4th Oct so maybe we just wait until then for this: https://www.python.org/dev/peps/pep-0619/#schedule

Reply:
Yes, please provide wheels for the rc. See https://github.com/numpy/numpy/issues/19630#issuecomment-898314636